Coupled-Cluster/spin_free_CC/polar.py

```diff
@@ -0,0 +1,84 @@
+# A simple Psi 4 script to compute CCSD from a RHF reference
+# Scipy and numpy python modules are required
+#
+# Algorithms were taken directly from Daniel Crawford's programming website:
+# http://sirius.chem.vt.edu/wiki/doku.php?id=crawdad:programming
+# Special thanks to Lori Burns for integral help
+#
+# Created by: Daniel G. A. Smith
+# Date: 7/29/14
+# License: GPL v3.0
+#
+
+import time
+import numpy as np
+from helper_CCENERGY import *
```

Comment:
I agree with you @dsirianni I would like to clean up the code as well. As you can see,
I was mostly concerned with getting the equations right. I would try to bring them all inside  
one class of helper_CC with different files for each helper class. Thanks for the feedback.